engine/gamesys/src/gamesys/resources/res_texture.h

```diff
@@ -17,6 +17,7 @@
 
 #include <resource/resource.h>
 #include <dmsdk/gamesys/resources/res_texture.h>
+#include <dmsdk/graphics/graphics.h>
```

Comment:
This header is not used?
In order to keep code dependencies as low as possible, and build times low, we want to remove unused headers from headers (and ofc cpp files)

Looks like the `res_texture.h` isn't used either?